### ISSUE TITLE
How to evaluate models of type AutoModelForSeqeuenceClassification?

### ISSUE THREAD

**Issue by YIYANGCAI** [FlagOpen/FlagEmbedding#289]
Hi,

Thanks for the great work and insights! 

I have the following question about finetuning: I directly use your code [eval code](https://github.com/FlagOpen/FlagEmbedding/blob/master/C_MTEB/eval_MTEB.py) eval public model **BAAI/bge-small-en-v1.5** and I can definitely get the right results. However, when I train the it on mteb/stsbenchmark-sts, I have to convert it to a AutoModelForSequenceClassification model, instead of AutoModel. How can I use your code to evaluate this AutoModelForSequenceClassification on the mteb/sts's other datasets? Since the model will output logits instead of embeddings. I am looking forward to your response. Thank you!

Best regards.
Frank.

**Comment by staoxiao** [comment]
Thanks for your interest in our work!
The eval code is developed based on MTEB, which doesn't support AutoModelForSequenceClassification. Maybe you can get some useful suggestions in the MTEB repo: https://github.com/embeddings-benchmark/mteb
